README

```diff
@@ -2,6 +2,9 @@
     :alt: Coverage Badge
     :target: https://coveralls.io/github/GeoNode/geonode?branch=master
```

Comment:
Why not remove the badge from coveralls.io?

Reply:
Thanks @gustry for the suggestion. I left it because I did not find any discussion in the dev list what are the benefits to switch to codecov. Meanwhile we can have both

Reply:
AFAIK, coveralls is not used anymore, no?

Reply:
For reference: https://github.com/GeoNode/geonode/pull/3364#discussion_r146863773

Reply:
Thanks for the reference. I’m personally inclined to having it approved from the dev community rather than the preferences of one single developer

Reply:
Thank's @francbartoli @Gustry . I'm fine with coveralls if it can support travis's matrix test, but from what I know coveralls doesn't support this (https://github.com/lemurheavy/coveralls-public/issues/156). When QGIS Server PR comes, it will use parallelized job on Travis to test each GeoServer and QGIS Server backend independently. Coveralls will report significantly lower coverage, because when it test GeoServer Backend, it will not execute QGIS Server Backend code and vice versa. That's the main reason I switched to codecov. Codecov will aggregate coverage from each parallel tests, so we have a total coverage. If this is a problem, then perhaps I can try reaching out through the dev community (through the dev mailing lists, I suppose?) to explain the reason why I proposed codecov.

Reply:
@lucernae thanks for your explanation. I'm fine with Codecov and much more happy to know that it can support both test cases. I think it's worth for geonode.
Can you just explain it through the dev mailing list so after the approval we can then definitively remove coveralls?

Reply:
Sure, I will send email soon.